### PR TITLE
Refactor code to conform with Coding Standards

### PR DESCRIPTION
**Pull request by ClFeSc**
I took some time and refactored the code base to conform with our [Coding Standards](https://github.com/hpi-swa-teaching/SketchMorph2/wiki/Coding-Standards). I addressed all points excluding
> Breaks in blocks, loops, etc.

and
> Alphabetical sorting of instance / class variables

I made a commit for each rule so we can individually review and/or remove them as necessary. I think that there will be many points which can be further improved, both regarding things I missed and things I did not quite right. I also don't want to build any pressure to review or even merge this PR soon, it should just be a starting point.

### CODE REVIEW

File: packages/SketchMorph2-Core.package/M2Backend.class/instance/rectangleBrushSize.st [hunk 1]
```diff
@@ -1,3 +1,3 @@
 accessing
 rectangleBrushSize
-	^ (0.5@1) * self brushSize
\ No newline at end of file
+	^ (0.5@1) * self brushSize
```
Comment (kpostnov) [inline packages/SketchMorph2-Core.package/M2Backend.class/instance/rectangleBrushSize.st]: Spaces could be added.

Comment (ClFeSc) [reply packages/SketchMorph2-Core.package/M2Backend.class/instance/rectangleBrushSize.st]: Good point. I'll look into it.

Comment (ClFeSc) [reply packages/SketchMorph2-Core.package/M2Backend.class/instance/rectangleBrushSize.st]: I added them in [5dc5732](https://github.com/hpi-swa-teaching/SketchMorph2/pull/73/commits/5dc573231db622f3500259b7b2351838614966a9).

**Comment by ClFeSc** [comment]
I'll no longer pursue this PR. It is very much diverged from `dev` since it was made before the Zoom. For most of it we now have #96, and I'm also not really happy with what I did here for things that #96 doesn't fix.

**Pull request closed without merging**